tests/unit/h5p.spec.js

```diff
@@ -51,6 +51,11 @@ describe('Component', () => {
     await flushPromises()
     expect(wrapper.vm.loading).toBe(false)
     expect(renderDefault).toHaveBeenCalledTimes(1)
+    expect(wrapper.vm.iframeLoading).toBe(true)
+    expect(renderDefault).toHaveBeenCalled()
+    wrapper.vm.iframeLoading = false
```

Comment:
So this is something that I'd like to do properly in general. Do be able to do this, jsdom needs to support iframes `srcdoc` - which it doesn't yet. I might want to look into creating a PR for that, but this will take a while.